policies/0008-contact-information-storage.md

```diff
@@ -0,0 +1,50 @@
+# Where to store contact information
+
+We will store all types of contact information here. It will include
```

Comment:
I propose `We will store information for technical contacts here.  It will include`

Reply:
done